machine_management/creating-infrastructure-machinesets.adoc

```diff
@@ -13,8 +13,6 @@ that are required to run the environment.
 
 include::modules/infrastructure-components.adoc[leveloffset=+1]
 
-include::modules/machine-api-overview.adoc[leveloffset=+1]
-
 [id="creating-infrastructure-machinesets-production"]
```

Comment:
Do we need `creating-infrastructure-machinesets.adoc` section at all?

>You can create a MachineSet to host only infrastructure components.
You apply specific Kubernetes labels to these Machines and then
update the infrastructure components to run on only those Machines.These
infrastructure nodes are not counted toward the total number of subscriptions
that are required to run the environment.

Is taht statement true?
cc @derekwaynecarr @eparis

Reply:
@enxebre, last I heard it was true. @derekwaynecarr, will you PTAL?

Making *a* new machineset and making new machinesets for your cluster infra are different user goals, so they need to be separate in our new doc structure.

Reply:
there isnt anything "special" about an infrastructure machine set.  its a machine set like any other.  i would orient discussion around infrastructure specific hosts with those components that would run on those hosts.

Reply:
Thanks, Derek! Since it's a different user goal, setting up an infrastructure machineset is in a different assembly, but the steps to create the machineset are identical. The infrastructure assembly additionally contains the specific steps to move workloads to the infrastructure machines. https://docs.openshift.com/container-platform/4.1/machine_management/creating-infrastructure-machinesets.html